modules/api/php/visitdicomsrowprovisioner.class.inc

```diff
@@ -0,0 +1,73 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all Dicoms
+ * of a visit.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api;
+
+use \LORIS\Data\Provisioners\DBRowProvisioner;
+/**
+ * This class implements a data provisioner to get all Dicoms
+ * from a Visit.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class VisitDicomsRowProvisioner extends DBRowProvisioner
+{
+    /**
+     * Create a RowProvisioner, which gets rows for a Visit Dicoms
+     *
+     * @param \Timepoint $timepoint The requested timepoint
+     */
+    function __construct(\Timepoint $timepoint)
+    {
+        parent::__construct(
+            '
+             SELECT
+               t.TarchiveID as tarchiveid
+             FROM tarchive t
+             JOIN session s
+               ON (s.ID=t.SessionID)
+             JOIN candidate c
+               ON (s.CandID=c.CandID)
+             WHERE 
+               c.Active="Y" AND
```

Comment:
From doc:

> A string is a sequence of bytes or characters, enclosed within either single quote (') or double quote (") characters.

> If the ANSI_QUOTES SQL mode is enabled, string literals can be quoted only within single quotation marks because a string quoted within double quotation marks is interpreted as an identifier. 

I will make a pass to change them.